MapboxCoreNavigation.podspec

```diff
@@ -2,12 +2,12 @@ Pod::Spec.new do |s|
 
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.name = "MapboxNavigation.swift"
-  s.version = "0.1.0"
+  s.name = "MapboxCoreNavigation"
+  s.version = "0.2.0"
   s.summary = "Mapbox Navigation library"
```

Comment:
The summary and description for this podspec should differ from the summary and description in the MapboxNavigation podspec.